corehq/form_processor/tests/test_basics.py

```diff
@@ -474,7 +485,43 @@ def test_caching_form_attachment_during_submission(self):
             _submit_case_block(True, uuid.uuid4().hex, user_id='user2', update={})
 
 
-def _submit_case_block(create, case_id, xmlns=SYSTEM_FORM_XMLNS, **kwargs):
+@flag_enabled('SYNC_SEARCH_CASE_CLAIM')
+@patch('corehq.motech.repeaters.models.domain_has_privilege', lambda x, y: True)
+class CaseSearchTests(FundamentalBaseTests):
+    def setUp(self):
+        super().setUp()
+        self.elasticsearch = get_es_new()
+        ensure_index_deleted(CASE_SEARCH_INDEX)
+        initialize_index_and_mapping(get_es_new(), CASE_SEARCH_INDEX_INFO)
+
+    def tearDown(self):
+        super().tearDown()
+        ensure_index_deleted(CASE_SEARCH_INDEX)
+
+    @patch.object(CouchUser, 'get_by_user_id', return_value=None)
+    @patch.object(Domain, 'get_by_name')
+    def test_create_case_and_update_elasticsearch(self, domain_mock, user_mock):
+        domain_mock.return_value = Domain(name=DOMAIN, web_apps_sync_case_search=True)
+
+        case_id = uuid.uuid4().hex
+        modified_on = datetime.utcnow()
+        xmlns = 'http://commcare.org/test_xmlns'
+        _submit_case_block(
+            True, case_id, user_id='user1', owner_id='owner1', case_type='demo',
+            case_name='create_case', date_modified=modified_on, date_opened=modified_on, update={
+                'dynamic': '123'
+            },
+            device_id=FORMPLAYER_DEVICE_ID,
+            xmlns=xmlns
+        )
+        self.elasticsearch.indices.refresh(CASE_SEARCH_INDEX)
```

Comment:
```suggestion
        # This case is sent to elasticsearch synchronously, without pillowtop
        self.elasticsearch.indices.refresh(CASE_SEARCH_INDEX)
```
I figure this could use a comment or something around specifically what's being tested